evap/staff/tools.py

```diff
@@ -294,8 +295,14 @@ def merge_users(main_user, other_user, preview=False):
 
 
 def find_unreviewed_evaluations(semester, excluded):
+    # as evaluations are open for an offset of hours after vote_end_datetime, the evaluations ending yesterday are also xcluded during offset
+    exclude_date = date.today()
+    if datetime.now().hour < settings.EVALUATION_END_OFFSET_HOURS:
+        exclude_date -= timedelta(days=1)
```

Comment:
@janno42 just making sure: There is no case we ever expect this setting to be more than 24 hours, right?

Reply:
right, that woudn't really make sense